src/Microsoft.Owin.Host.HttpListener/OwinHttpListener.cs

```diff
@@ -213,6 +213,12 @@ private async void ProcessRequestsAsync()
                 try
                 {
                     context = await _listener.GetContextAsync();
+                    if (IsEmptyPayloadAndContentLength(context))
```

Comment:
Rather than sniffing for certain types of requests (we're always going to miss some) I'd rather harden the error handling to prevent a cash. But first we need to track down the original exception it was trying to handle when the ODE was thrown.